.github/workflows/build.yaml

```diff
@@ -173,7 +173,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04]
-        buildType: [RelWithDebInfo]
+        buildType: [Debug]
```

Comment:
Using a Debug build makes the CI run maybe ~1 minute longer ;P Users are not running debug builds in production, therefore it would make sense to me to test mostly with RelWithDebInfo.

I think there should be maybe one debug build in the CI, not all of them. BTW, Coverity warns if there are C asserts that have side-effects, so I presume debug builds only serve to introduce some timing randomness into tests...